[2021-08-23T17:43:57.825730+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/","method":"GET"} []
[2021-08-23T17:43:57.828285+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-23T17:43:57.828337+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T17:43:57.828364+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T17:43:57.875841+02:00] doctrine.DEBUG: SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1 [1] []
[2021-08-23T17:43:57.886337+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1' with params [1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list'" at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1' with params [1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:112)"} []
[2021-08-23T17:43:57.964966+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-23T17:43:58.111254+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/","method":"GET"} []
[2021-08-23T17:43:58.118073+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-23T17:43:58.118121+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T17:43:58.118146+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T17:43:58.150091+02:00] doctrine.DEBUG: SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1 [1] []
[2021-08-23T17:43:58.159603+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1' with params [1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list'" at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1' with params [1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:112)"} []
[2021-08-23T17:43:58.255105+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-23T17:43:58.685074+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8ba185"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/_wdt/8ba185","method":"GET"} []
[2021-08-23T18:07:05.462869+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/","method":"GET"} []
[2021-08-23T18:07:05.464627+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-23T18:07:05.464684+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:07:05.464713+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:07:05.494206+02:00] doctrine.DEBUG: SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ? [1] []
[2021-08-23T18:07:05.500792+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list'" at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:112)"} []
[2021-08-23T18:07:05.581818+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-23T18:07:06.031534+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"864e7d"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/_wdt/864e7d","method":"GET"} []
[2021-08-23T18:07:09.864918+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/","method":"GET"} []
[2021-08-23T18:07:09.871381+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-23T18:07:09.871441+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:07:09.871470+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:07:09.908444+02:00] doctrine.DEBUG: SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ? [1] []
[2021-08-23T18:07:09.917830+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list'" at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:112)"} []
[2021-08-23T18:07:10.001885+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-23T18:07:10.426344+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"fde672"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/_wdt/fde672","method":"GET"} []
[2021-08-23T18:12:28.740660+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/","method":"GET"} []
[2021-08-23T18:12:28.747696+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-23T18:12:28.747751+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:12:28.747778+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:12:28.785689+02:00] doctrine.DEBUG: SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ? [1] []
[2021-08-23T18:12:28.795998+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list'" at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:112)"} []
[2021-08-23T18:12:28.883832+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-23T18:12:29.327693+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"519e0d"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/_wdt/519e0d","method":"GET"} []
[2021-08-23T18:12:31.608676+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/","method":"GET"} []
[2021-08-23T18:12:31.614651+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-23T18:12:31.614703+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:12:31.614729+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:12:31.650542+02:00] doctrine.DEBUG: SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ? [1] []
[2021-08-23T18:12:31.667446+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list'" at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT i0_.id AS id_0, i0_.image AS image_1, i0_.banner AS banner_2, i0_.thumbnail AS thumbnail_3, i0_.article_id AS article_id_4 FROM image i0_ WHERE i0_.banner = ?' with params [1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'i0_.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:112)"} []
[2021-08-23T18:12:31.755491+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-23T18:12:32.203959+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a79f42"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/_wdt/a79f42","method":"GET"} []
[2021-08-23T18:12:58.967204+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/","method":"GET"} []
[2021-08-23T18:12:58.968734+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-23T18:12:58.968788+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:12:58.968816+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-23T18:12:58.986266+02:00] doctrine.DEBUG: SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1 [1] []
[2021-08-23T18:12:58.992420+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1' with params [1]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list'" at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 79 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.image AS image_2, t0.banner AS banner_3, t0.thumbnail AS thumbnail_4, t0.article_id AS article_id_5 FROM image t0 WHERE t0.banner = ? LIMIT 1' with params [1]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:79)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.banner' in 'field list' at /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:112)"} []
[2021-08-23T18:12:59.065877+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-23T18:12:59.491907+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3fa4ba"},"request_uri":"http://bs9otamvav.preview.infomaniak.website/_wdt/3fa4ba","method":"GET"} []
[2021-08-24T19:42:55.890867+02:00] php.DEBUG: Warning: filemtime(): stat failed for /home/clients/260820d70569867bbcdb59e4dad3e521/sites/wild-inside.fr/config/routes/dev/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/home/letour/Dev/Client/WildInside/vendor/symfony/config/Resource/FileResource.php","line":62,"trace":[{"file":"/home/letour/Dev/Client/WildInside/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-08-24T19:44:26.685881+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"https://127.0.0.1:8000/","method":"GET"} []
[2021-08-24T19:44:26.695773+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-24T19:44:26.695819+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-24T19:44:26.695841+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-24T19:44:26.739546+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in driver: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost'" at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 128 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:128)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:39)"} []
[2021-08-24T19:44:26.839483+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-24T19:44:27.134287+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"174c8c"},"request_uri":"https://127.0.0.1:8000/_wdt/174c8c","method":"GET"} []
[2021-08-24T19:44:46.272838+02:00] request.INFO: Matched route "home". {"route":"home","route_parameters":{"_route":"home","_controller":"App\\Controller\\HomeController::index"},"request_uri":"https://127.0.0.1:8000/","method":"GET"} []
[2021-08-24T19:44:46.273549+02:00] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2021-08-24T19:44:46.273593+02:00] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-24T19:44:46.273614+02:00] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"main","authenticator":"App\\Security\\LoginFormAuthenticator"} []
[2021-08-24T19:44:46.276308+02:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in driver: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost'" at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 128 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:128)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\n[previous exception] [object] (PDOException(code: 1698): SQLSTATE[HY000] [1698] Access denied for user 'root'@'localhost' at /home/letour/Dev/Client/WildInside/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:39)"} []
[2021-08-24T19:44:46.348800+02:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-08-24T19:44:46.655144+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"211d33"},"request_uri":"https://127.0.0.1:8000/_wdt/211d33","method":"GET"} []
